### DISCUSSION TITLE
Project Tools!

### DISCUSSION THREAD

**Discussion by shannonrankin** [nmfs-ost/PAM-OpenScience#12] [Polls]
There are a number of tools for organizing tasks in a larger project, which is your favorite?  Feel free to reply with pros/cons of other tools!

**Comment by rebecca-vanhoeck** [comment]
We are migrating all of our NEFSC project management to Smartsheets. Sadly moving away from Monday.

**Reply by shannonrankin** [reply]
Would love to learn why, and how smartsheets works for you (compared w/ Monday)

**Reply by rebecca-vanhoeck** [reply]
I don't think I can change my vote, or vote again. We're still very much in the migration phase, but so far smartsheets is working really well!

**Comment by jlkeating** [comment]
I use a combination of Google Sheets (new table platform) and Github projects.

**Comment by emmonsca** [comment]
We use meetings and tasks in google docs mostly due to simplicity

**Comment by cberchok** [comment]
We use emails, excels, word docs, google docs.
